.travis.yml

```diff
@@ -11,28 +11,28 @@ matrix:
   - rust: nightly
     before_script:
     - rustup component add rustfmt-preview
-    - rustup target add wasm32-unknown-unknown
+    - cargo install cargo-web
     script:
     - cargo fmt --all -- --check
     - cargo test --features "serde std v1 v3 v4 v5"
     - cargo bench --features "serde std v1 v3 v4 v5"
-    - cargo build --target wasm32-unknown-unknown --features "v3 stdweb"
-    - cargo build --target wasm32-unknown-unknown --features "v4 stdweb"
-    - cargo build --target wasm32-unknown-unknown --features "v5 stdweb"
-    - cargo build --target wasm32-unknown-unknown --features "v3 wasm-bindgen"
-    - cargo build --target wasm32-unknown-unknown --features "v4 wasm-bindgen"
-    - cargo build --target wasm32-unknown-unknown --features "v5 wasm-bindgen"
+    - cargo web build --features "v3 stdweb"
```

Comment:
Do you know what a simple `cargo web build` does differently than `cargo build --target wasm32-unknown-unknown`?

Reply:
Apparently there are hidden switches that are flipped around with cargo-web

The build failure particularly mentioned using it and it works